packages/server-wallet/src/message-service/test-message-service.ts

```diff
@@ -165,3 +188,9 @@ function formatMessageForLogger(message: Message) {
     requests: data.requests?.map(r => `${r.type}-${r.channelId}`),
   };
 }
+
+export function isTestMessageService(
+  messageService: MessageServiceInterface
+): messageService is TestMessageService {
+  return '_frozen' in messageService;
```

Comment:
This seems a little cryptic :)

Reply:
I think I should probably add a more explicit field to narrow the type on.